### ISSUE TITLE
Alacritty Fails to draw update on MacBook

### ISSUE THREAD

**Issue by Derrekito** [alacritty/alacritty#8210]
Alacritty frequently fails to update the drawn window. 

For example, if I type "ls" the screen may display "l" and if I type another "s" the window will display "lss". 

### System

OS: macOS
Version: 0.13.2

### Logs

NO crashes. I am using tmux.

**Comment by chrisduerr** [comment]
What's your hardware? macOS OpenGL drivers are pretty bad, so this might just be an upstream driver issue.

**Comment by Derrekito** [comment]
MacBook Pro 2019

2.4 Ghz, 8-core Inte Core i9

Intel UHD Graphics 630

macOS 15.0

**Comment by chrisduerr** [comment]
Is this only an issue with macOS 15? That release has been extremely buggy so I wouldn't rely on it functioning properly.

**Comment by kchibisov** [comment]
Given that it's an intel platform, I wouldn't be surprised if they broke something on it with OpenGL.

**Comment by chrisduerr** [comment]
I wouldn't even be surprised if they did so *intentionally*, which is why a test against the previous macOs version makes sense.

**Comment by AdamAdamsMusic** [comment]
Wanted to chime in as I started having this issue yesterday – upgraded from macOS Monterey (12.7.4) to Ventura (13.7.3) and it started happening. 
Can confirm this only happens in Alacritty - haven't seen it in both Terminal and iTerm2. 

Coincidentally, I have also updated to the latest Alacritty version (0.14), but even rolling back to 0.13.2 does not fix the issue, so I guess this must be on the macOS side. 

MacBook Pro 2017
2.4 Ghz, 4-core Intel Core i5
Intel UHD Graphics 630
